python/tvm/relay/frontend/onnx.py

```diff
@@ -4978,10 +4978,30 @@ def _impl_v1(cls, inputs, attr, params):
         return _expr.TupleWrapper(_expr.Tuple(result), len(result))
 
 
+class Round(OnnxOpConverter):
+    """Operator converter for round op."""
+
+    @classmethod
+    def _impl_v11(cls, inputs, attr, params):
+        # See the tutorial for full implementation details:
+        # https://www.youtube.com/watch?v=fZvNG-oKK5Q&list=PL_4zDggB-DBpynCEnC9hV-1euZrP3xDRK&index=4
```

Comment:
It would be better to explain what onnx is doing (and why we need to handle it specially) in the comment rather than linking to youtube.

Reply:
If you want, you can just credit josh in the PR text if this is about credit

Reply:
@jwfromm Do you feel that the line 4989 doesn't give enough context in regards to what onnx is doing? 

@AndrewZhaoLuo This was more about having the value of Josh's tutorial accessible.

Reply:
@jwfromm Perhaps flipping the comments so that like 4989 comes before the youtube link is better readability?

Reply:
I personally think the code is simple enough where the implementation details are self-documenting

Reply:
sounds good :)